response_operations_ui/controllers/notify_controller.py

```diff
@@ -39,14 +37,16 @@ def _send_message(self, email, template_id, personalisation=None, reference=None
             notification.update({"reference": reference})
 
         url = f'{self.notify_url}{template_id}'
-
         auth = app.config['SECURITY_USER_NAME'], app.config['SECURITY_USER_PASSWORD']
         response = requests.post(url, auth=auth, json=notification)
-        if response.status_code == 201:
+        status_code = response.status_code
+
+        if 201 <= status_code <= 399:
```

Comment:
The only valid responses from notify-gateway are 201 and 400 [source](https://github.com/ONSdigital/rm-notify-gateway/blob/master/API.md)

```suggestion
        if status_code == 201:
```